src/components/Progress/Progress.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Progress } from 'reactstrap';
+
+import './Progress.scss';
+
+export function ASABProgress({
+	value = 0,
```

Comment:
@JanMajer86 I dont think you need to specify anything except `...props` and `children` here.

```
ASABProgress({ children, ...props }) { ...
```